packages/server/src/db/index.js

```diff
@@ -323,7 +322,11 @@ async function getGrants({
                     queryBuilder.orderBy(`${TABLES.grants}.grant_id`, orderArgs[1]);
                 } else {
                     const orderArgs = orderBy.split('|');
-                    queryBuilder.orderBy(...orderArgs);
+                    const orderDirection = ((ascending === 'true') ? 'asc' : 'desc');
+                    if (orderArgs.length > 1) {
+                        console.log(`Too many orderArgs: ${orderArgs}`);
+                    }
```

Comment:
Why are we enforcing only a single sort field now? I doubt the frontend will provide UI for specifying multiple but I could imagine having a default tiebreaker.

Reply:
I'm reluctant to implement a feature which hasn't been requested, and (in my opinion) is open-ended enough that it's hard to create a comprehensive suite of automated tests.